### PR TITLE
Add markers docs

### PR DESCRIPTION
**Pull request by chris-wood-mat**

### CODE REVIEW

File: docs/manual/articles/markers.md [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+- End Timestamp \- An optional point in the session where this range of interest ends. If not specified the end of the session is implied.
+
+!!! tip
+    To create a marker which represents a specific point in time the start and end time properties can be set to the same value.
+
+Each marker can have any number of labels associated with it to provide additional custom information. Marker labels have the following properties:
+
+- Name \- A short, computer readable name for referencing the label in code.
+- Label \- A short, human readable summary of what the label represents.
+- Description \- A longer form human readable explanation of what the label represents.
```
Comment (stanislav-domaratsky-mclaren) [inline docs/manual/articles/markers.md:21]: "A longer form" might not be the best wording. How about "A detailed, human readable explanation..."?

File: docs/manual/articles/markers.md [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+- Value \- A value associated with this marker.
+- Format \- The C-Style format string used when displaying the value to the user e.g. `%.3f` for a real number displayed to 3 decimal places.
+- Unit \- The definition of the units that the value is measured in.
+
+None of these properties are required and they can be used in any combination e.g. using only the label or using just the value and units.
```
Comment (stanislav-domaratsky-mclaren) [inline docs/manual/articles/markers.md:26]: I'd better say "None of these properties are **mandatory**"

File: docs/manual/articles/markers.md [hunk 3]
```diff
@@ -0,0 +1,68 @@
...
+!!! example
+    ```c#
+    var marker = new Marker(markerStartTime, markerEndTime, "High Wind", "windSpeedData",
+        "A period during which the wind was sustained above 9 m/s");
+
+    marker.AddLabel(
+        new MarkerLabel
+        {
+            Label = "Average Speed",
+            Description = "A average sustained wind speed during this time period",
```
Comment (stanislav-domaratsky-mclaren) [inline docs/manual/articles/markers.md:46]: "**An** average sustained wind speed"

**Review by stanislav-domaratsky-mclaren — CHANGES_REQUESTED** [review]

**Comment by stanislav-domaratsky-mclaren** [comment]
Looks good!

**Review by colled — APPROVED** [review]

**Pull request merged**